src/components/pod/pod.spec.js

```diff
@@ -834,6 +1013,14 @@ function renderEditAction(props = {}, renderer = mount) {
   return renderer(<StyledEditAction {...props} />);
 }
 
+function renderDeleteAction(props = {}, renderer = mount) {
+  return renderer(<StyledDeleteButton {...props} />);
```

Comment:
These two new functions throw errors in the console when running the tests when no `onAction` or `children` are passed